packages/lifecycle/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "pagopa-interop-lifecycle",
```

Comment:
```suggestion
  "name": "pagopa-interop-agreement-lifecycle",
```

Reply:
@galales Wouldn't it be more convenient to maintain a generic lifecycle package with entity-specific folders (eg: Agreement) ? This way, if we need to share lifecycle logic again for any reason, we already have a package ready for that purpose without needing to create new ones. Compared to the past, using the monorepo makes it easier to share logic like this.

Reply:
let's make it specific for now, and change it to generic if/when required in the future

Reply:
Done with commit [77a9dba6d8de129c9cc82b412136a11a7862a829](https://github.com/pagopa/interop-be-monorepo/pull/698/commits/77a9dba6d8de129c9cc82b412136a11a7862a829)